docs/src/pages/core/number-input.md

```diff
@@ -0,0 +1,137 @@
+---
+title: NumberInput
+group: 'svelteuidev-core'
+packageGroup: '@svelteuidev/core'
+slug: /core/number-input/
+category: 'inputs'
+description: 'Number input with optional controls'
+import: "import { NumberInput } from '@svelteuidev/core';"
+source: 'svelte-core/src/lib/components/NumberInput/NumberInput.svelte'
+docs: 'core/number-input.md'
+---
+
+<script>
+    import { Box } from '@svelteuidev/core';
+    import { Prism } from '@svelteuidev/prism';
+    import { Demo, NumberInputDemos } from '@svelteuidev/demos';
+    import { CodeBlock, Heading, Preview } from 'components';
+
+    const controlled = `
+    <script>
+        import { NumberInput } from '@svelteuidev/core';
+
+        let value;
+        function onChange(e) {
+            value = e.detail;
+        }
+    <\/script>
+
+    <NumberInput value={value} on:change={onChange} \/>
+    `;
+
+    const blur = `<NumberInput noClampOnBlur \/>`;
+
+    const accessibility = `
+    <NumberInput /> // -> not ok, input is not labeled
+    <NumberInput label="Your age" /> // -> ok, input and label are connected
+    <NumberInput aria-label="Your age" /> // -> ok, label is not visible but will be announced by screen reader
+    `;
+
+    let value;
+    function onChange(e) {
+        value = e.detail;
+    }
+</script>
+
+<style global>
+    input {
+        margin: 0;
+    }
+</style>
+
+<Heading />
+
+## Usage
+
+<Demo demo={NumberInputDemos.configurator} />
+
+## Controlled
+
+<Box css={{ pre: { bc: '$gray50' }, 'pre code': { color: '$gray900' } }}>
+    <Prism language='svelte' code={controlled} />
+</Box>
+
```

Comment:
- Why did you opt to use Prism instead of the markdown code block?
- It's okay to call it controlled but in the example you do it the react way. For example you can simply `bind:value` and it achieves the same effect.